src/Mutation.php

```diff
@@ -173,14 +160,17 @@ public function getOriginalFileAst(): array
         return $this->originalFileAst;
     }
 
-    public function isOnFunctionSignature(): bool
+    /**
+     * @return CoverageLineData[]
+     */
+    public function getAllTests(): array
     {
-        return $this->isOnFunctionSignature;
+        return $this->tests;
     }
 
     public function isCoveredByTest(): bool
     {
-        return $this->isCoveredByTest;
+        return \count($this->getAllTests()) !== 0;
```

Comment:
let's witch to a `use` statement :) I know it's better to fix it with a CS fixer, but I think we can already change it for newly introduced/changed code